src/components/tx/TxStepper/useTxStepper.ts

```diff
@@ -32,12 +31,10 @@ export const useTxStepper = ({ steps, initialData, initialStep, onClose, onFinis
 
   const handleNext = () => {
     setActiveStep((prevActiveStep) => prevActiveStep + 1)
-    trackEvent({ category: MODALS_CATEGORY, action: lastStep ? 'Submit' : 'Next' })
   }
 
   const handleBack = () => {
     setActiveStep((prevActiveStep) => prevActiveStep - 1)
-    trackEvent({ category: MODALS_CATEGORY, action: firstStep ? 'Cancel' : 'Back' })
```

Comment:
My understanding is, these two events weren't in safe-react and weren't documented.
I think they are two generic to track. It's gonna be hard to analyze them. Removed.

Reply:
The equivalent [is in `safe-react`](https://github.com/safe-global/safe-react/blob/bccf8b3259c5b175fd61972bea49c0261327a0ba/src/routes/safe/components/Transactions/helpers/TxModalWrapper/index.tsx#L333). It can be used in funnels.

Reply:
True, will restore them then. The only problem the stepper is also used by Create Safe as per Franco's findings.

Reply:
They are useful there. We can analyse if a certain step is 'successful', i.e. the user doesn't tend to go back.

Reply:
They might be, but the name is "modals"

Reply:
I see. Perhaps we should create a new category for "generic" values then?

Reply:
I think the way to go would be to pass `category: 'create-safe'` and keep `modals` for modals.

Reply:
Sounds good!

Reply:
Added a todo in the issue.